RestSharpApi/Tests/MilestonesTests.cs

```diff
@@ -0,0 +1,144 @@
+        {
+            Authenticator = new HttpBasicAuthenticator("atrostyanko@gmail.com", "Qwertyu_1")
+        };
+
+        Milestone milestone = new Milestone
+        {
+            Name = "Makusheva Test Milestone",
+            Description = "Test Makusheva"
+        };
+
+        // Setup Rest Client
+        var client = new RestClient(options);
+
+        // Setup Request
+        var request = new RestRequest(endpoint)
+            .AddUrlSegment("project_id", 6)
+            .AddJsonBody(milestone);
+
+        var response = client.ExecutePost<Milestone>(request);
+        Milestone actualProject = response.Data;
+
+        Logger.Info(actualProject);
+
+        Assert.Multiple(() =>
+        {
+            Assert.That(response.StatusCode, Is.EqualTo(HttpStatusCode.OK));
+
+            Assert.That(actualProject.Name, Is.EqualTo(milestone.Name));
+
+            Assert.That(actualProject.Equals(milestone));
+        });
+
+    }
+
+
+    [Test]
+    [Order(2)]
+    public void MilestonesGetTest()
+    {
+        const string endpoint = "index.php?/api/v2/get_milestones/{project_id}";
+
+        var options = new RestClientOptions(BaseRestUri)
+        {
+            Authenticator = new HttpBasicAuthenticator("atrostyanko@gmail.com", "Qwertyu_1")
+        };
+            
+        // Setup Rest Client
+        var client = new RestClient(options);
+
+        // Setup Request
+        var request = new RestRequest(endpoint)
+            .AddUrlSegment("project_id", 6);
+
+        // Execute Request
+        var response = client.ExecuteGet(request);
+        
+        Logger.Info(response.Content);
+        
+        Assert.That(response.StatusCode, Is.EqualTo(HttpStatusCode.OK));
+    }
+
+    [Test]
+    [Order(3)]
+    public void MilestoneUpdateTest()
+    {
+        const string endpoint = "index.php?/api/v2/update_milestone/{milestone_id}";
+
+        var options = new RestClientOptions(BaseRestUri)
+        {
+            Authenticator = new HttpBasicAuthenticator("atrostyanko@gmail.com", "Qwertyu_1")
+        };
+
+        Milestone milestone = new Milestone
+        {
+            Name = "NEW Test Description Makusheva"
+        };
+
+        // Setup Rest Client
+        var client = new RestClient(options);
+
+        // Setup Request
+        var request = new RestRequest(endpoint)
+            .AddUrlSegment("milestone_id", 1)
+            .AddJsonBody(milestone);
+
+        var response = client.ExecutePost<Milestone>(request);
+        Milestone actualProject = response.Data;
+
+        Logger.Info(actualProject);
+
+        Assert.Multiple(() =>
+        {
+            Assert.That(response.StatusCode, Is.EqualTo(HttpStatusCode.OK));
+
+            Assert.That(actualProject.Equals(milestone));
+        });
+    }
+
+    [Test]
+    [Order(4)]
+    public void MilestoneDeleteTest()
+    {
+        const string endpoint = "index.php?/api/v2/delete_milestone/{milestone_id}";
```

Comment:
Assert.That(response.StatusCode, Is.EqualTo(HttpStatusCode.OK))
  Expected: OK
  But was:  BadRequest